File: apps/webserv/www/cgi-bin/hello.php
<?php
header("Content-Type: text/plain; charset=utf-8");
header("X-CGI-Runtime: php");

echo "Hello from PHP-CGI\n";
echo "method=" . ($_SERVER["REQUEST_METHOD"] ?? "") . "\n";
echo "query=" . ($_SERVER["QUERY_STRING"] ?? "") . "\n";
?>

File: www/cgi-bin/hello.py
import os
import sys


sys.stdout.write("Content-Type: text/plain; charset=utf-8\r\n")
sys.stdout.write("\r\n")
sys.stdout.write("Hello from webserv CGI\n")
sys.stdout.write("method=" + os.environ.get("REQUEST_METHOD", "") + "\n")
